tasks/request_status_for_granule/request_status_for_granule.py

```diff
@@ -183,8 +182,8 @@ def get_job_entry_for_granule_sql() -> text:
                 SELECT
                     granule_id as "{OUTPUT_GRANULE_ID_KEY}",
                     job_id as "{OUTPUT_JOB_ID_KEY}",
-                    EXTRACT(EPOCH FROM request_time AT TIME ZONE 'UTC') * 1000 as "{OUTPUT_REQUEST_TIME_KEY}",
-                    EXTRACT(EPOCH FROM completion_time AT TIME ZONE 'UTC') * 1000 as "{OUTPUT_COMPLETION_TIME_KEY}"
+                    EXTRACT(EPOCH FROM date_trunc('milliseconds', request_time) AT TIME ZONE 'UTC') * 1000 as "{OUTPUT_REQUEST_TIME_KEY}",
```

Comment:
You will want to cast the EXTRACT to a bigint datatype so that the python driver will express it as an int datatype instead of a float.
`(EXTRACT(EPOCH FROM date_trunc('milliseconds', request_time) AT TIME ZONE 'UTC') * 1000)::bingint as "{OUTPUT_REQUEST_TIME_KEY}"`

Reply:
Whoops, missed. Thanks.